checker/src/org/checkerframework/checker/regex/RegexUtil.java

```diff
@@ -2,16 +2,16 @@
 
 package org.checkerframework.checker.regex;
 
-import java.util.regex.Pattern;
```

Comment:
This was an unused import when I made the pull request. I think trunk already removed this, too.